aip/0123.md

```diff
@@ -73,8 +74,64 @@ message Topic {
 - Pattern variables (the segments within braces) **must** use `snake_case`, and
   **must not** use an `_id` suffix.
 
+#### Complex resource ID path segments
+
+Note: Complex resource ID path segments are supported for historical reasons,
```

Comment:
Let's use `**Warning:**`. Note the bold-facing, which is important to trigger the callout.

Reply:
Done